src/s3b_the_roach_market.tw2

```diff
@@ -0,0 +1,280 @@
+<!-- If using debug menu shortcuts, prompts player to enter their name before beginning initial Roach Market sequence -->
+::RM-first-entrance-nameprompt
+{ (set: $playerName to (prompt: "Enter name:", ""))
+(if: (text: $playerName)'s length > 1)[Right, my name is $playerName. [[Moving on|RM-first-entrance]].<!-- what this code does is check the ending of the custom $playerName string to hopefully pluralize it properly -->{
```

Comment:
I made this same mistake on my code — this should actually be `length > 0`, right?

Reply:
Yeah I think one-letter names are fine! I stuck with the logic from your code in case there was any reason to not want that.